core/management/commands/launch.py

```diff
@@ -138,7 +138,7 @@ def launch_server( command, **options ):
 		safe_print( 'Launching RFID server thread...' )
 		for k, v in kwargs.items():
 			safe_print( '    {}={}'.format( k, v if isinstance(v, (int,float)) else '"{}"'.format(v) ) )
-		thread = threading.Thread( target=runServer, kwargs=kwargs, name='LLRPServer'. daemon=True )
+		thread = threading.Thread( target=runServer, kwargs=kwargs, name='LLRPServer', daemon=True )
```

Comment:
Thanks for the change. However, this was corrected recently in a development release. Not sure if @esitarski pushed it to a prod release.

Reply:
Glad someone chimed in on the mailing list to help the user then... https://groups.google.com/g/crossmgrsoftware/c/0t1pR7spQjY/m/ZMLKUWPLAQAJ